bin/reformat_spec.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
Is this Windows friendly?

Reply:
I think Windows doesn't care about shebangs... SO seems to confirm: https://stackoverflow.com/questions/7574453/shebang-notation-python-scripts-on-windows-and-linux
so this would be of help on OSX/*nix and of no (positive/negative) effect on windows

Reply:
Why is this one `python3` and the other one `python`? Shouldn't the environment dictate the version?

Reply:
Eh, forgot detail. May be because this one isn't python 2 compatible. I am ok to change to regular python here